common/web/input-processor/src/text/prediction/predictionContext.ts

```diff
@@ -80,6 +80,8 @@ export default class PredictionContext extends EventEmitter<PredictionContextEve
     this.suggestionApplier = (suggestion) => {
       if(validSuggestionState()) {
         return langProcessor.applySuggestion(suggestion, this.currentTarget, () => kbdProcessor.layerId);
+      } else {
+        return Promise.resolve(null);
```

Comment:
Is this a bug fix?

Reply:
`noImplicitReturns` strikes again.  Before, we implicitly returned `Promise.resolve(undefined)`.

Reply:
That's actually not true. Before, we implicitly returned `undefined`, because this is not an `async` function.

So, I also did an audit of this one. This change introduces a new bug:

https://github.com/keymanapp/keyman/blob/92ffb33c79c08188b3582fbfbacac4b9c323f746/common/web/input-processor/src/text/prediction/predictionContext.ts#L189-L190

The test on L190 above will never be `true` now, because `revertAcceptancePromise` will always be a `Promise`, not nullish.